git_pile/genbranch_caching.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, path, committer_ident=None):
         self.__cache_data = _CacheData.load(self.__path)
 
         if not committer_ident:
-            committer_ident = _git(["var", "GIT_COMMITTER_IDENT"]).stdout
```

Comment:
maybe do `import ... as h` to make it shorter?

Reply:
Not a big fan of having a one-letter identifier to represent a global thing.

Reply:
ok... sometimes I prefer using `as h` for it to be clear it's coming from another module.... may be better than polluting this namespace if you do  `from .helpers import git`. As long as we are consistent throughout the code, I don't care much